### PR TITLE
test(ideahub):added ideahub tests

### PR DESCRIPTION
**Pull request by mudit94**
# Closes/Fixes/Resolves

added ideahub tests

# Explain the feature/fix

added ideahub tests- migrating  idea hub tests 
## Does this PR introduce a breaking change
no

## Screenshots

<details>
<summary>View Screenshots</summary>

</details>

### Ready-for-merge Checklist

- [ ] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [ ] Does the change have appropriate unit tests?
- [ ] Did tests pass?
- [ ] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
- [ ] Was this feature demo'd and the design review approved?

### CODE REVIEW

File: e2e/helper/apps.list.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+];
+
+export const microservicesDesc = [
+    "Adding authentication to your application is pice of cake. Just integrate with our API and focus on application development.",
+    "Allows users to review and rate services and products provided by teams. Built into a power packed microservice so serve all.",
+    "Notification framework for One Platform and also all other SPAs",
+    "A prebuilt integration with LDAP to authenticate, authorize and get user information.",
+    "A unified search API gives you ability to search across all hosted applications on the platform."
+];
+
```
Comment (riginoommen) [inline e2e/helper/apps.list.js:28]: WHy this content is needed. this is specific to one platform. i suggest to do a clean up

File: e2e/helper/apps.list.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+export const apps = [
+    "Blog",
+    "/components",
+    "/get-started/docs/faqs",
+    "/feedback",
+    "/get-started",
+    "/",
+    "#",
```
Comment (riginoommen) [inline e2e/helper/apps.list.js:60]: Include the Required details for the idea hub specific

File: cypress.env.example.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
+    "STAGE_HOST":"",
```
Comment (riginoommen) [inline cypress.env.example.json:2]:
```suggestion
    "QA_HOST":"",
```

you mentioned QA_HOST in scripts update the env variables

File: cypress.env.example.json [hunk 4]
```diff
@@ -0,0 +1,6 @@
...
+    "USERNAME": "",
+    "PASSWORD": "",
+    "JIRA":""
```
Comment (riginoommen) [inline cypress.env.example.json:5]:
```suggestion
```

**Review by riginoommen — CHANGES_REQUESTED** [review]

**Review by ghanlohar — APPROVED** [review]
LGTM

**Review by riginoommen — APPROVED** [review]

**Review by akhilmhdh — APPROVED** [review]

**Pull request merged**